### ISSUE TITLE
Can't log in / reset password

### ISSUE THREAD

**Issue by LiamMacmillan** [VivaPayments/API#1451]
Trying to reset my password (not on demo). I receive a text message with a code. After entering it, I am prompted to reset my password. However, on submit, the text message popup appears again with an "INCORRECT CODE" error. I am not sent a new code, and the previous one does not work. Essentially, I am unable to access my account.
<img width="1440" alt="Screenshot 2020-03-10 at 18 05 57" src="https://user-images.githubusercontent.com/54713960/76344648-d3528200-62f9-11ea-849d-72aa097c2b9d.png">

**Comment by matthewrgourd** [comment]
Hi there, sorry to hear this. Please send me your Merchant ID and business name quoting this issue (#1451) and I will investigate. My email address is mgourd@vivawallet.com. Thanks!

**Comment by matthewrgourd** [comment]
Hi @LiamMacmillan thanks for sending through your account info. I've checked and everything appears to be in order. The only thing I noticed is that your email is unconfirmed. It might be worth checking in your spam as we would have sent an email asking you to verify your email. As regards the password reset facility, I just tested using my own account and there seems to be no issue. Sorry not to be of more help. I'll update you if I think of anything, in the meantime you might want to call our customer service team (+44) 20 3734 7770 and ask them if they can do a password reset on your behalf. Thanks...
